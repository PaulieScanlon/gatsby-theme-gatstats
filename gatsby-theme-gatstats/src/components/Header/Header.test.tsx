import * as React from 'react'
import { shallow } from 'enzyme'

import { ThemeProvider } from 'theme-ui'
import theme from '../../gatsby-plugin-theme-ui'

import { Header } from './'

describe('<Header>', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <ThemeProvider theme={theme}>
        <Header sideBarWidth={0} pathname="/about-me">
          children
        </Header>
      </ThemeProvider>
    )
    expect(wrapper).toMatchSnapshot()
  })
})
