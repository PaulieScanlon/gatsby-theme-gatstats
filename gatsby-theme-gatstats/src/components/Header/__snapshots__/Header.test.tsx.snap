// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> renders correctly 1`] = `
<RootProvider
  theme={
    Object {
      "colors": Object {
        "background": "#fff",
        "modes": Object {
          "dark": Object {
            "background": "#000",
            "primary": "#0cf",
            "text": "#fff",
          },
        },
        "primary": "#07c",
        "text": "#000",
      },
      "fontSizes": Array [
        12,
        14,
        16,
        20,
        24,
        32,
        48,
        64,
        72,
      ],
      "fontWeights": Object {
        "body": 400,
        "bold": 700,
        "heading": 700,
      },
      "fonts": Object {
        "body": "system-ui, sans-serif",
        "heading": "system-ui, sans-serif",
        "monospace": "Menlo, monospace",
      },
      "initialColorModeName": "light",
      "lineHeights": Object {
        "body": 1.5,
        "heading": 1.125,
      },
    }
  }
>
  <Header>
    children
  </Header>
</RootProvider>
`;
